Dockerfile

```diff
@@ -0,0 +1,38 @@
+ARG GO_VERSION="1.22"
+ARG ALPINE_VERSION="3.19"
```

Comment:
```suggestion
ARG ALPINE_VERSION="3.19.1"
```

Reply:
yes, I know... but there is no tag [alpine3.19.1](https://hub.docker.com/_/golang/tags?page=1&name=alpine3.19) for golang available

and if you check the image details [golang:1.22-alpine3.19](https://hub.docker.com/layers/library/golang/1.22-alpine3.19/images/sha256-5a99b4049412cd34ad6b4e0c9527ae6beb9ae82d787b4bf3f4eff7aa13fc577a?context=explore) it's using underneath **alpine:3, 3.19, 3.19.1, latest**

I think it's common practice in docker image tagging, when you leave out the `patch` part of the version, that `major.minor` should be the same as `major.minor`.*latest-available-patch*